packages/mui-material/src/styles/createPalette.ts

```diff
@@ -9,6 +9,147 @@ import blue from '../colors/blue';
 import lightBlue from '../colors/lightBlue';
 import green from '../colors/green';
 
+export type PaletteMode = 'light' | 'dark';
+export interface Color {
+  50: string;
+  100: string;
+  200: string;
+  300: string;
+  400: string;
+  500: string;
+  600: string;
+  700: string;
+  800: string;
+  900: string;
+  A100: string;
+  A200: string;
+  A400: string;
+  A700: string;
+}
+
+export {};
```

Comment:
Does this have a purpose?

Reply:
I think we can remove this. Don't see the point of having it.

Reply:
removed here https://github.com/mui/material-ui/pull/44173/commits/5819055bf9ef4e5247fc05d99f72db17f84a4162

Reply:
On second thoughts, [many files does have `export {}` in their `d.ts` files](https://github.com/search?q=repo%3Amui%2Fmaterial-ui%20export%20%7B%7D&type=code). it looks like this is used to disabled automatic export. shall i revert it back?

https://github.com/mui/material-ui/blob/452d971330cd5e7f85c24b23e91ef7567bb3c25d/packages/mui-types/index.d.ts#L3-L4

Reply:
What do you mean by disabling automatic export?